src/site/content/en/blog/min-max-clamp/index.md

```diff
@@ -0,0 +1,179 @@
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./caniuse.png" alt="" />
+  <figcaption class="w-figcaption">
+    <a href="https://caniuse.com/?search=min%20max%20clamp">Caniuse</a> support table.
+  </figcaption>
+</figure>
+
+## Usage
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/min-demo.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Showing how the min() function selects a value based on a list of options and its parent. <a href="https://codepen.io/una/pen/rNeGNVL">See Demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+You can use `min()`, `max()`, and `clamp()` on the right hand side of any CSS expression where it would make sense. For `min()` and `max()`, you provide an argument list of values, and the browser determines which one is either the smallest or largest, respectively. For example, in the case of: `min(1rem, 50%, 10vw)`, the browser calculates which of these relative units is the smallest, and uses that value as the actual value.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/max-demo.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Showing how the max() function selects a value based on a list of options and its parent. <a href="https://codepen.io/una/pen/RwaZXqR">See Demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+
+The `max()` function selects the largest value from a list of comma-separated expressions.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/clamp-demo.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Showing how the clamp() function selects a value based on a list of options and its parent. <a href="https://codepen.io/una/pen/bGpoGdJ">See Demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+
+To use `clamp()` enter three values: a minimum value, ideal value (from which to calculate), and maximum value.
+
+Any of these functions can be used anywhere a `<length>`, `<frequency>`, `<angle>`, `<time>`, `<percentage>`, `<number>`, or `<integer>` is allowed. You can use these on their own (i.e. `font-size: max(0.5vw, 50%, 2rem)`), in conjunction with `calc()` (i.e. `font-size: max(calc(0.5vw - 1em), 2rem)`),  or composed (i.e. `font-size: max(min(0.5vw, 1em), 2rem)`). 
+
+<aside>
+When using a calculation inside of a `min()`, `max()`, or `clamp()` function, you can nix the `calc()`. For example, writing `font-size: max(calc(0.5vw - 1em), 2rem)` would be the same as `font-size: max(0.5vw - 1em, 2rem)`.
+</aside>
```

Comment:
```suggestion
{% Aside %}
When using a calculation inside of a `min()`, `max()`, or `clamp()` function, you can nix the `calc()`. For example, writing `font-size: max(calc(0.5vw - 1em), 2rem)` would be the same as `font-size: max(0.5vw - 1em, 2rem)`.
{% endAside %}
```
This wasn't displaying correctly because 1) you can't use Markdown inside of HTML and 2) we don't use the built-in aside element, we have a [custom component](https://web.dev/handbook/web-dev-components/#note-asides)